package.json

```diff
@@ -141,6 +141,14 @@
     "webpack": "^5.68.0",
     "webpack-dev-server": "^3.11.2"
   },
+  "peerDependencies": {
+    "typescript": "4.2.x"
```

Comment:
Since we are about to merge #1073 (path parameter inference), I fail to remember which version of TypeScript added that feature. 

- [4.2 Release notes](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-2.html)
- [4.3 Release notes](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-3.html)

Reply:
The pull request description mentions 4.1. Great!